src/assets/styles/colours.scss

```diff
@@ -2,3 +2,22 @@
 $hel-coat: #0072c6;
 $hel-copper: #00d7a7;
 $hel-suomenlinna: #f5a3c7;
+
+$hel-bus: #0000bf;
+$hel-bus-light-20: #ccccf2;
+$hel-bus-light-50: #7f7fdf;
+$hel-bus-dark-50: #000098;
+
+/* GRAY SHADES FOR UI COMPONENTS */
```

Comment:
Should we use these variables at all?

Reply:
are these in hds?

Reply:
Yeah, we could refactor these out from the project. Maybe separate PR for that